priv/repo/migrations/20160918003206_fix_mixed_case_slugs.exs

```diff
@@ -0,0 +1,24 @@
+defmodule CodeCorps.Repo.Migrations.FixMixedCaseSlugs do
+  use Ecto.Migration
+
+  alias CodeCorps.Repo
+  alias CodeCorps.SluggedRoute
+
+  import CodeCorps.ModelHelpers
+
+  def up do
+    SluggedRoute
+    |> Repo.all
+    |> Repo.preload([:user, :organization])
+    |> Enum.each(fn record ->
+      changeset =
+        SluggedRoute.changeset(record)
+        |> Ecto.Changeset.put_change(:slug, Inflex.parameterize(record.slug))
+        |> Repo.update!
+    end)
+  end
```

Comment:
Could do a separate method here, then call it like 
```Elixir
|>Enum.each(&parametrize_slug(&1)
```

I don't think that's really necessary in a migration, though.